### PR TITLE
Add a contributing file

### PR DESCRIPTION
**Pull request by jaredgalanis**
This is a WIP. We need to fill in the template and figure out what additional content we want to add, including what to say about naming of files, ids and links.

### CODE REVIEW

**Comment by jaredgalanis** [comment]
We're seeking to merge this even though it is currently a WIP just so we have something that can be built on.

**Review by MinThaMie — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+TODO: Figure out exactly what to tell people about file names, id's and links.
+
+For guidance on the structure of a deprecation file, please see `deprecation-template.md`.
+
+## Making Your First Pull Request
+
+First-time contributors are encouraged to choose issues that are labeled
+"help wanted" or "good for new contributors." If you have questions, want a
+suggestion of what to work on, or would like a buddy to pair with, you can
+join the #-team-learning channel in the
```
Comment (MinThaMie) [inline CONTRIBUTING.md:18]:
```suggestion
join the #dev-ember-learning channel in the
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Fork this repository (click "fork" on the repository's home page in GitHub)
+
+Clone the forked repository with `git clone <your fork's url>` and create a
+branch with `git checkout -b some-branch-name`.
+
+As you make commits, reference the issue number in your commit message, such as
+`git commit -m "add glasses to nearsighted hamster (#6217)"`
+
+Once you're at the point that you'd like feedback, submit a Pull Request (new
+Pull Request button). Choose `master` for the base and your branch name for `compare`,
```
Comment (MinThaMie) [inline CONTRIBUTING.md:30]:
```suggestion
Pull Request button). Choose `main` for the base and your branch name for `compare`,
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+then submit it!
+
+Your PR will be reviewed by another contributor, and then either merged or have
+changes requested.
+
+## Keeping your fork updated
+
+As you tackle new Issues, you'll want to be sure that you always start by working
+on the most recent code. To sync up your fork's  `master` with a parent repository's
```
Comment (MinThaMie) [inline CONTRIBUTING.md:39]:
```suggestion
on the most recent code. To sync up your fork's  `main` with a parent repository's
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+master, set an upstream and pull from it. For this to work, you should make sure
```
Comment (MinThaMie) [inline CONTRIBUTING.md:40]:
```suggestion
main, set an upstream and pull from it. For this to work, you should make sure
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+you're always committing to a branch, not master.
```
Comment (MinThaMie) [inline CONTRIBUTING.md:41]:
```suggestion
you're always committing to a branch, not main.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+git remote add upstream https://github.com/ember-learn/guides-source.git
+git checkout master
```
Comment (MinThaMie) [inline CONTRIBUTING.md:45]:
```suggestion
git checkout main
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+git pull upstream master
```
Comment (MinThaMie) [inline CONTRIBUTING.md:46]:
```suggestion
git pull upstream main
```

**Pull request merged**